tests/lib/spread/rules/feature-tagging.yaml

```diff
@@ -0,0 +1,18 @@
+rules:
```

Comment:
Can you please start with a comment that links to an explanation of what this file is for and what reads it?